demos/helpers/colorSelector.js

```diff
@@ -13,19 +13,19 @@ define(function(require) {
      * @return {void}
      */
     function createColorSelector(selectContainerSelector, chartSelector, callback) {
-        var colorKeys = Object.keys(colors.colorSchemas),
+        let colorKeys = Object.keys(colors.colorSchemas),
```

Comment:
I think here `colorKeys` and `containerSelector` can be both constants since they are not being changed in that function. Would rewrite it as:
```
const colorKeys = Object.keys(colors.colorSchemas)
const containerSelector = document.querySelector(selectContainerSelector);
```

Reply:
Yeah, doing it.